apps/block_scout_web/lib/block_scout_web/views/transaction_view.ex

```diff
@@ -584,4 +589,33 @@ defmodule BlockScoutWeb.TransactionView do
   defp template_to_string(template) when is_tuple(template) do
     safe_to_string(template)
   end
+
+  # Function decodes revert reason of the transaction
```

Comment:
this should be `@doc` instead of a comment - the IDE can retrieve documentation for methods if the `@doc` attribute is used correctly